client/components/popup.jsx

```diff
@@ -32,13 +33,73 @@ export default class PopUp extends React.Component {
       const flag = this.context.map((values, i) => {
         if (values[0] === flightCiao) {
           if (values[2] === "United States") {
```

Comment:
great work putting such a complex conditional statement together! Now it is time to make it dynamic!

I would recommend only doing the first if check for values[0], and then using the values[2] to dynamically create the src and alt text.